tests/e2e/utils/experimentalFeatures.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Toggle experiments state.
+ *
+ * @param {Array<string>}features Array of experiments to enable.
+ * @param {boolean} enable Whether the features should be enabled or not.
+ * @return {Promise<void>}
+ */
+async function toggleExperiments(features, enable) {
```

Comment:
FWIW, this util allows us to write tests for things that are still behind flags.